### PR TITLE
Challenge Completion enhancement

### PR DESCRIPTION
**Pull request by benalleng**
This is a further enhancement of #58 .

green is good!
image

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Tested. This feels like a step in the right direction. However I also wonder what it would like to have the challenge completion (celebration) animated as well.
Screenshot 2023-02-01 at 9 22 58 PM

**Review by afterburn — APPROVED** [review]
image

I think we can get this merged and do the animation in a separate pr when we make a decision on how to proceed.

**Comment by ecurrencyhodler** [comment]
LOL that meme is amazing. Okay looks like this PR is still in draft mode.

File: ui/common/StatusBar.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default function StatusBar({
       className={clsx('border-t border-white/25 max-md:px-4 max-md:py-8', {
         'w-screen': full,
         'w-full': !full,
-        'bg-green-400/25': getStatus() === Status.Success,
+        'bg-[#28b12326]': getStatus() === Status.Success,
```
Comment (afterburn) [inline ui/common/StatusBar.tsx:65]: Are we planning to use this color for other things apart from this element? If so, I think its better to move this to our `tailwind.config.js`. If it's a one off, ignore my comment.

Comment (benalleng) [reply ui/common/StatusBar.tsx]: I decided to narrow the focus of this PR to just handle the success status bar color, We can continue to consider the custom tailwind config but for now lets just make sure we see green

**Comment by ecurrencyhodler** [comment]
Tested again. Looks good and ran into no bugs for me.

**Comment by afterburn** [comment]
Same here, lgtm

**Pull request merged**